templates/interstitial.html

```diff
@@ -6,17 +6,17 @@
 
 {% block form_content %}
   {% if block['section_number'] %}
-    <h1 class="u-fs-l">Section {{ block['section_number'] }}: {{ block['title'] }}</h1>
+    <h1>Section {{ block['section_number'] }}: {{ block['title'] }}</h1>
```

Comment:
These are removed in another open PR, need to ensure we merge properly to ensure your changes are accounted for.